tests/main.cpp

```diff
@@ -5,47 +5,46 @@
 
 #include <iostream>
 #include <mlboard/core.hpp>
-#include <unistd.h>
 #include <dirent.h>
```

Comment:
These two header files are not there in windows (Or to be specific visual studio ) so the code won't work

Note : it is there in mingw